src/main/java/org/galatea/starter/service/IexService.java

```diff
@@ -69,6 +79,96 @@ private String setKey() {
     }
   }
 
+  /**
+   * Formats a date from YYYYMMDD format to YYYY-MM-DD format. This is to accommodate the historical
+   * price repository which automatically does this conversion.
+   * @param date A string representing a date in YYYYMMDD format.
+   * @return A string representing the same date in YYYY-MM-DD format.
+   */
+
+  private String formatDate(final String date) {
```

Comment:
Does this method NEED to be in the service class? Is there another package that may be better suited for it?